src/azure-account.ts

```diff
@@ -3,213 +3,40 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-// eslint-disable-next-line @typescript-eslint/no-var-requires, @typescript-eslint/no-unsafe-assignment
```

Comment:
How does the resulting vsix compare after your changes? Specifically, I want to make sure the result of a webpack looks about the same (same number of files, same approximate size of the files) so that we're not affecting performance

Also when you say "I made sure the manual tests are passing for these changes" were you F5'ing or running against the webpacked vsix? Please make sure to at least spot check with the webpacked vsix

Reply:
This PR's vsix is 755k compared to 754k before the changes. And the old & new vsixes have the same number of files.

I was F5ing but I just verified the test cases with the vsix as well. I had to include a new dev dependency that vsce complained about.

Reply:
I don't think renaming files would related to dev dependencies at all. Seems fishy

Reply:
This was the first time I tried running `vsce package`. I get the same error on main